modules/integration_aws-redshift/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,13 @@
+module: AWS Redshift
+name: heartbeat
+
+transformation: false
+aggregation: ".mean(by=['ClusterIdentifier', 'NodeID'])"
```

Comment:
Should we check if all nodes respond or only the cluster itself ?
I don't have the answer I honestly do not know redshift.

Reply:
You're right. I need to check the cluster only.